cmd/evicttestcli/main.go

```diff
@@ -0,0 +1,302 @@
+package main
```

Comment:
where is this file coming from ? is it for manual tests ? why not use it under the relevant package and incorporate that as a unittests or functional tests ?